gen_matrix.py

```diff
@@ -0,0 +1,68 @@
+import collections
+import operator
+from pathlib import Path
+
+import mkdocs_gen_files
+import tabulate
+import tomli
+
+import ibis
+import ibis.expr.operations as ops
+
+
+def get_backends():
+    pyproject = tomli.loads(Path("pyproject.toml").read_text())
+    backends = pyproject["tool"]["poetry"]["plugins"]["ibis.backends"]
+    return [
+        (backend, getattr(ibis, backend))
+        for backend in sorted(backends.keys())
+    ]
+
+
+def get_leaf_classes(op):
+    for child_class in op.__subclasses__():
+        if not child_class.__subclasses__():
+            yield child_class
+        else:
+            yield from get_leaf_classes(child_class)
+
+
+with mkdocs_gen_files.open(
+    Path("backends") / "support_matrix.md",
+    "w",
+) as f:
+    print("---", file=f)
+    print("hide:", file=f)
+    print("  - toc", file=f)
+    print("---", file=f)
+    print(file=f)
+    print("# Operation Support Matrix", file=f)
+    print(file=f)
+    print(
+        '!!! note "Backends are shown in descending order of the number of supported operations"',  # noqa: E501
+        file=f,
+    )
+    print(file=f)
+    print('<div class="support-matrix" markdown>', file=f)
```

Comment:
I wonder, could we [feed an existing plugin](https://github.com/timvink/mkdocs-table-reader-plugin) with a csv instead?

Reply:
Great idea, will look into it!

Reply:
This worked out pretty nicely. I'm now using the plugin to read the file.